src/styles/settings/_colors.scss

```diff
@@ -0,0 +1 @@
+$sangre: red;
```

Comment:
When naming variables in Sass, I like to name them based on what role the color plays in your app, not based on the color itself. For example `$primaryColor` and `$accentColor` instead of `$skyblue` or `$periwinkle`. This way, if I want to update the colors on my site, I can do so without making the variable names irrelevant. Think about a site that may change color schemes based on the time of year (spring colors --> fall colors, for example).